docs/integration.rst

```diff
@@ -171,6 +171,8 @@ Note that build systems may handle parts of these dependencies automatically,
 but also in this case you will have to select some of them manually as RAUC
 cannot fully know how you intend to use your system.
 
+:General: unsquashfs (from `Squashfs
+          <http://squashfs.sourceforge.net/>`_)
```

Comment:
I fear this mixes things that should not be mixed. The other tools are required to extract or install the update on the target. Instead, `unsquashfs` is neither used nor required for this.

The only situations where unsquashfs is used are calls on `rauc info` (for extracting the manifest without unpacking the bundle)  and `rauc convert` (for converting it into a casync bundle). During normal extraction, the bundle only becomes mounted to extract the actual images.